merkle/src/merkle.rs

```diff
@@ -211,6 +238,17 @@ impl<E: Element> Store<E> for MmapStore<E> {
         self.len += 1;
     }
 
+    fn write_range(&mut self, buf: &[u8], start: usize) {
+        let b = E::byte_len();
+        assert_eq!(buf.len() % b, 0);
+        let r = std::ops::Range {
+            start: start * b,
+            end: start * b + buf.len(),
+        };
+        self.store[r].copy_from_slice(buf);
```

Comment:
you can just write `self.store[start * b..start * b + buf.len()]` no need to construct a range manually

Reply:
also technically `write_slice` or `copy_from_slice` would describe it a bit better

Reply:
oh, nice, good to know